lib/elixir/lib/calendar.ex

```diff
@@ -465,6 +465,7 @@ defmodule Calendar do
   P      | "am" or "pm" (noon is "pm", midnight as "am")                           | am, pm
   q      | Quarter                                                                 | 1, 2, 3, 4
   S      | Second                                                                  | 00, 59, 60
+  s      | The number of seconds since the Epoch, 1970-01-01 00:00:00+0000 (UTC)   | 1565888877
```

Comment:
```suggestion
  s      | The number of seconds since the Epoch, 1970-01-01 00:00:00+0000 (UTC)   | 1565888877
  S      | Second                                                                  | 00, 59, 60
```